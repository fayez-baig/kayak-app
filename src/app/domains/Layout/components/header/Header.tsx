import { FC, ReactElement } from "react";
import { SvgIcon } from "app/domains/Common/components/svg-icon";
import "./header.scss";

const Header: FC = (): ReactElement => {
  return (
    <header className="header">
      <SvgIcon icon="KayakLogoIcon" />
    </header>
  );
};

export default Header;
