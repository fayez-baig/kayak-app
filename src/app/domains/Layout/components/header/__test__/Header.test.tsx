import { createRenderer } from "react-test-renderer/shallow";

import Header from "../Header";

const renderer = createRenderer();

describe("<Header />", () => {
  it("should render and match the snapshot", () => {
    renderer.render(<Header />);
    const renderedOutput = renderer.getRenderOutput();
    expect(renderedOutput).toMatchSnapshot();
  });
});
