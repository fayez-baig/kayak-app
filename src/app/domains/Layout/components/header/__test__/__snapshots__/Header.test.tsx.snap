// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> should render and match the snapshot 1`] = `
<header
  className="header"
>
  <SvgIcon
    icon="KayakLogoIcon"
  />
</header>
`;
